tests/unit_tests/test_suite_test.go

```diff
@@ -0,0 +1,91 @@
+package tests
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"kubevirt.io/client-go/kubecli"
+
+	instancetypev1beta1 "kubevirt.io/api/instancetype/v1beta1"
+)
+
+const (
+	testNamespace                  = "common-instancetype-functest"
+	clusterInstanceTypesBundlePath = "../../common-clusterinstancetypes-bundle.yaml"
+	clusterPreferencesBundlePath   = "../../common-clusterpreferences-bundle.yaml"
```

Comment:
I'm not sure, but wouldn't be safer to use an absolute path here?

Reply:
IMO it's fine for now.

Reply:
I will drop this, it is not needed anymore. We will require to have the resources deployed before running tests